Tools/CISupport/ews-app/ews-app/settings.py

```diff
@@ -35,7 +35,7 @@
 import ews.common.util as util
 
 is_test_mode_enabled = util.load_password('EWS_PRODUCTION') is None
-is_dev_instance = (util.load_password('DEV_INSTANCE', default='').lower() == 'true')
+is_dev_instance = (util.get_custom_suffix() != '')
```

Comment:
Nit: Could we just do `util.get_custom_suffix()` since '' evaluates as false?

Reply:
That would work as well. I wanted to be explicit here since various other things (like None, False) can also evaluate to False, and the get _custom_suffix() explicitly returns '' for prod instance.